test/test_visualization.jl

```diff
@@ -4,18 +4,16 @@ using Test
 using DispersiveShallowWater
 using Plots
 
-@testset "Unit tests" begin
-    @testset "Visualization" begin
-        trixi_include(@__MODULE__, default_example(), tspan = (0.0, 1.0))
-        @test_nowarn plot(semi => sol)
-        @test_nowarn plot!(semi => sol, plot_initial = true)
-        @test_nowarn plot(semi, sol, conversion = prim2cons, plot_bathymetry = false)
-        @test_nowarn plot(semi => sol, 0.0)
-        @test_nowarn plot(semi, sol, 0.0, conversion = prim2cons)
-        @test_nowarn plot(analysis_callback)
-        @test_nowarn plot(analysis_callback, what = (:errors,))
-        @test_nowarn plot(analysis_callback, what = (:integrals, :errors))
-    end
+@testset "Visualization" begin
```

Comment:
This is unrelated, but I just noticed the additional `@testset "Unit tests"` is not really helpful here.